internal/cli/cmd/dac/diff/diff.go

```diff
@@ -82,34 +83,36 @@ func (o *option) Validate() error {
 }
 
 func (o *option) Execute() error {
+	// Create the output folder (+ any parent folder if applicable) where to store the diff files
+	err := os.MkdirAll(config.Global.Dac.OutputFolder, os.ModePerm)
+	if err != nil {
+		return fmt.Errorf("error creating the output folder: %v", err)
+	}
+
+	var execResult strings.Builder
 	for _, preview := range o.dashboards {
 		project := resource.GetProject(preview.GetMetadata(), o.Project)
 		dashboard, err := o.apiClient.V1().Dashboard(project).Get(preview.Metadata.Name)
```

Comment:
I thought one of the idea was to generate the diff file if the there is no previous version. Would be less miss-leading if we don't display an error in case previous dashboard is not found (since it's a new dasbhoard)

Reply:
we didnt discuss that case afaik but good idea indeed, 'need to see how to cleanly generate such diff (it should full of "+" lines then)